### ISSUE TITLE
Illustration of pretrained model checkpoint

### ISSUE THREAD

**Issue by irleader** [instadeepai/InstaNovo#37]
Hi,

Changelog for release 0.1.4 states:  "add checkpoints instanovo.pt trained on HC-PT, and instanovo_yeast.pt fine-tuned on nine-species excluding yeast."

After checking the vocab/residues for both model checkpoints:
instanovo.pt does not have 'N(+.98)', 'Q(+.98)', and uses 'C' and 'M(ox)'
instanovo_yeast.pt has 'N(+.98)', 'Q(+.98)', and uses 'C(+57.02)' and 'M(+15.99)'.

Therefore, I doubt instanovo_yeast.pt can not be finetuned based on instanovo.pt as they have different vocabs/residues.

So is instanovo_yeast.pt trained from scratch on nine-species dataset excluding yeast?

Thanks!

**Comment by irleader** [comment]
Also, is it possible to share withus the  instanovo.pt checkpoint trained on HC-PT and has 'N(+.98)', 'Q(+.98)', and uses 'C(+57.02)' and 'M(+15.99)'?

**Comment by KevinEloff** [comment]
Hi @irleader,

Apologies again for the delayed response.

We actually do use the same checkpoint, but modify the decoder to allow for the new vocabulary. In the [1.0.0 release](https://github.com/instadeepai/InstaNovo/releases/tag/1.0.0) this process is performed automatically, adjusting the size of the amino acid embedding layer, the head weight, and head bias of the decoder.

In the 0.1.4 release, we did this manually by removing the weights associated with the input and output of the decoder and trained them from scratch on the nine-species dataset. We did this for both InstaNovo and InstaNovo+.
